yb-voyager/cmd/importData.go

```diff
@@ -594,18 +604,20 @@ func importData(importFileTasks []*ImportFileTask) {
 
 			useTaskPicker := utils.GetEnvAsBool("USE_TASK_PICKER_FOR_IMPORT", true)
 			if useTaskPicker {
-				err := importTasksViaTaskPicker(pendingTasks, state, progressReporter, poolSize, maxTasksInProgress)
+				maxColocatedBatchesInProgress := utils.GetEnvAsInt("MAX_COLOCATED_BATCHES_IN_PROGRESS", 3)
+				err := importTasksViaTaskPicker(pendingTasks, state, progressReporter, maxParallelConns, maxParallelConns, maxColocatedBatchesInProgress)
```

Comment:
passing `maxParallelConns` twice?

Reply:
yes haha 
- one is used to determine the size of the worker pool
- one is used to pass the maxShardedTasksInProgress. 

I suppose I could pass only once and infer `maxShardedTasksInProgress`, but i wanted to keep it as an arg of the function.